### DISCUSSION TITLE
Update SONOFF ZBDongle-E key

### DISCUSSION THREAD

**Discussion by alexxiiis-7** [darkxst/silabs-firmware-builder#119] [Q&A] [answered]
Good morning;
I would like to ask you a question about my SONOFF ZBDongle-E key.
I wanted to update it with the solution (https://darkxst.github.io/silabs-firmware-builder/), but it offers me the latest version "Zigbee (EZSP) 7.3.1.0 build 176".
On the Github Sonoff (https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E) version 7.4.3 is available.
If 7.4.3 is not offered it is that it is not yet stable or simply time to change the programming on your side?
Thank you for this information!

**Comment by sorryusernameisalreadytaken** [comment]
I have almost the same question to @darkxst 

Did you update the flasher by-time or should we just grab the latest build. For example I took this ncp fw for my sonoff dongle:
https://raw.githubusercontent.com/darkxst/silabs-firmware-builder/main/firmware_builds/zbdonglee/ot-rcp-v2.4.3.0-zbdonglee-460800.gbl
This results in:
**OpenThread (RCP) SL-OPENTHREAD/2.4.3.0_GitHub-7074a43e4**
web-flashers default RCP image seems to be:
**OpenThread (RCP) SL-OPENTHREAD/2.3.1.0_GitHub-e6df00dd6**

**Comment by darkxst** [comment, marked as answer]
yes 7.4.3 is stable, just haven't updated web flasher. 

For Thread 2.4.2 build is bundled in the OTBR addon for ZBDongle-E (via auto-flash firmware option)

There were no changes affecting Zigbee 7.4.3 or thread 2.4.3 so these are actually the same as x.4.2 builds

**Reply by alexxiiis-7** [reply]
Ok, in this case, to update to 7.4.3, I can do it by selecting "Change Firmware" then "upload your own firmware".
If so, what would be the source of the file to take?
I find several Github with several different versions and I do not want to be wrong.
Thank you very much!

**Reply by darkxst** [reply]
yes you can do that.
https://github.com/darkxst/silabs-firmware-builder/raw/main/firmware_builds/zbdonglee/ncp-uart-hw-v7.4.3.0-zbdonglee-115200.gbl

**Reply by alexxiiis-7** [reply]
Great! thank you very much!
What is the difference between the 115200 and 230400 versions?

**Reply by darkxst** [reply]
just stick with 115200. 230400 may help for very large and chatty networks but probably the extra speed doesnt make a difference in most cases.

**Reply by alexxiiis-7** [reply]
Great! thank you very much!
